nestris_ocr/colors.py

```diff
@@ -0,0 +1,83 @@
+from typing import Tuple
+import numpy as np
+from nestris_ocr.utils.lib import ilerp
+
+Color = Tuple[int, int, int]
+
+BLACKISH = (10, 10, 10)
+WHITEISH = (245, 245, 245)
+
+REFERENCE_LEVEL_COLORS = (
```

Comment:
Eventually I would like it if players could calibrate level colors by having players scroll through levels 0-9 and placing a few blocks in each.  For now this table is ok, but eventually we want to do something like save and load from a json configuration file.

Reply:
Aye, that will make  calibration a little long and painful.

I extracted the color table here from an emulator output, so it should work ok as a reference table. I reckon having a loadable color table can be in another PR later.

Reply:
Main problem with emulator colors is they don't reflect capture card values. If you want a good reference table, i suggest looking at this:
http://emulation.gametechwiki.com/index.php/Famicom_Color_Palette

Unsaturated-final v3 is pretty good, but default color palette of most emulator's is "RGB", (i don't know where your source is from) which pretty much only matches RGB modded consoles. We could also scrape all those palettes and let people choose from 2-3 presets (i think we'd only need unsaturated-final, composite, and RGB). However capture cards also distort it with brightness/gamma/contrast. EUGH.

I suspect the only levels *actually* affected would be 6 (in case grey is close to black), and 7 (as magenta/purple might be close to each other or black). The rest of the levels there's enough leniency with "pick closest of 4" that it probably doesn't matter.

I agree calibrating per level would be annoying though. The way i envisioned it would be that it grabs the blocks from the preview window + level text; since we already have code to determine piece and hence color. Then you'd only need 4-5 pieces per level, the bottleneck is restarting the game ;(

Reply:
Ooooh, that rabbit hole is deep 😰 

Thanks for the reference!